rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -1034,6 +1036,7 @@ public void onSuccess(String result) {
                                               nodeSourceJsonItem.setValue(result);
                                               nodeSourceJsonForm.submit();
                                               window.hide();
```

Comment:
Is this hide() still needed if you call destroy()?

Reply:
From the doc, `hide` hides in the DOM representation and `destroy` destroys the underlying JavaScript object, so I will keep both, thank you for making me look it up!